ModelViewer/ModelViewer/Axes.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+static const char* flatVertexShaderSource =
+"attribute highp vec4 posAttr;\n"
+"attribute lowp vec4 colAttr;\n"
+"varying lowp vec4 col;\n"
+"uniform highp mat4 matrix;\n"
+"void main() {\n"
+"   col = colAttr;\n"
+"   gl_Position = matrix * posAttr;\n"
+"}\n";
+
+static const char* flatFragmentShaderSource =
+"varying lowp vec4 col;\n"
+"void main() {\n"
+"   gl_FragColor = col;\n"
+"}\n";
+
+
+static const GLfloat vertices[] = {
+     0.0f,  0.707f,
+    -0.5f, -0.5f,
+     0.5f, -0.5f
+};
+
+static const GLfloat colors[] = {
+    1.0f, 0.0f, 0.0f,
+    0.0f, 1.0f, 0.0f,
+    0.0f, 0.0f, 1.0f
+};
+
+static const GLfloat grid[] = {
```

Comment:
You think it's worthwhile to allow the user to have a grid of a set size? Could be another issue. 

At the very least I'm going to add a 'toggle grid' option to #80 :)

Reply:
Oh yeah, this was a very primitive implementation of the grid.  I'm planning to add auto-resizing of the grid based on zoom level.